.buildkite/scripts/setenv.sh

```diff
@@ -2,19 +2,28 @@
 
 set -euo pipefail
 
+source .buildkite/scripts/common.sh
```

Comment:
are we adding this, so that we can call `add_bin_path` later?

If yes, I'd suggest that we don't add it here, but instead, make the necessary changes (update PATH, add inside `set-k8s-env.sh` (which, as the name implies, sets the environment up for k8s use) and then just `source .buildkite/deploy/kubernetes/scripts/setup-k8s-env.sh` (rather than call it directly) so that the PATH changes get applied.